modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/js/api/FieldBase/ReactFieldBase.js

```diff
@@ -413,6 +425,22 @@ export default function FieldBase({
 		[dispatch, editingLanguageId, pages]
 	);
 
+	useEffect(() => {
+		if (disableFieldRepetition) {
+			setDisabledRepeatableButton(true);
+		}
+		else {
+			Liferay.on('disableRepeatableButton', disableRepeatableButton);
+
+			return () => {
+				Liferay.detach(
+					'disableRepeatableButton',
+					disableRepeatableButton
+				);
+			};
+		}
```

Comment:
hey @rtienpal, I don't believe this condition is relevant for this specific case, can you check? Since we'll always go into the first condition if `disableFieldRepetition` is true